website/sidebars.js

```diff
@@ -33,13 +38,39 @@ module.exports = {
       ],
```

Comment:
The link above will need to be modified every time we release a new version of the SDK. I'm not sure what the best approach is for this, it would be good to discuss.